### ISSUE TITLE
session returned from `getIronSessionFromCookieStore` does not implement full interface

### ISSUE THREAD

**Issue by brumm** [vvo/iron-session#689]
Hey there,

We're using Nextjs's `cookies()` function. The session object returned from `getIronSessionFromCookieStore` does not include an implementation for `updateConfig`.

Our use case for `updateConfig` is to set a maxAge on the session that is not known until after logging in.

We've worked around this for now by allowing the `getSession` wrapper around `getIronSession` to pass new sessionOptions

```ts
export async function getSession(
  config: Partial<SessionOptions> = sessionOptions,
) {
  const session = getIronSession<SessionData>(
    cookies(),
    merge({}, sessionOptions, config),
  )
  return session
}

```

**Comment by simpros** [comment]
Same here - i am using v8.0.1

**Comment by vvo** [comment]
Fixes in 8.0.2
